### PR TITLE
Resnet18 train function + CE loss

### PR DESCRIPTION
**Pull request by rlindsberg**
Training with 20% data for 3 epochs:

image

image

PS.
If you cannot debug with Pycharm, because it freezes on "collecting data" etc. Check 'Gevent Compatible'.
(https://intellij-support.jetbrains.com/hc/en-us/community/posts/205819689-How-to-deal-with-collecting-data-in-debugger-?page=1#community_comment_360002651959)

Skærmbillede 2021-05-05 kl  17 40 16

### CODE REVIEW

**Review by YinengWang — APPROVED** [review]
In this [post](https://github.com/kuangliu/pytorch-cifar/issues/19), somebody came up with a better choice of normalization values: (0.4914, 0.4822, 0.4465), (0.247, 0.243, 0.261). We can experiment with these different choices.

**Comment by rlindsberg** [comment]
> In this [post](https://github.com/kuangliu/pytorch-cifar/issues/19), somebody came up with a better choice of normalization values: (0.4914, 0.4822, 0.4465), (0.247, 0.243, 0.261). We can experiment with these different choices.

Absolutely!
